config/class-site-details-index.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace Automattic\VIP\Config;
+
+class Site_Details_Index {
+	private static $_instance;
+
+	private $timestamp = null;
+
+	/**
+	 * Standard singleton except accept a timestamp for mocking purposes
+	 *
+	 * @param {mixed} $timestamp A fixed point in time to use for mocking
```

Comment:
Unlike JSDoc, the use of `{}` around `@param` and `@return` types is not standard for PHPDoc.

https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#abnf

For WPCS, summary and tag descriptions should also end with a full stop.

https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods

```suggestion
	 * @param mixed $timestamp A fixed point in time to use for mocking.
```